ppr-ui/src/composables/mhrInformation/useMhrInformation.ts

```diff
@@ -0,0 +1,60 @@
+import { useGetters } from 'vuex-composition-helpers'
+import {
+  MhrRegistrationDescriptionIF,
+  MhrRegistrationHomeLocationIF,
+  MhrRegistrationHomeOwnerGroupIF,
+  MhrRegistrationIF,
+  NewMhrRegistrationApiIF,
+  MhrTransferIF
+} from '@/interfaces'
+import { StaffPaymentIF } from '@bcrs-shared-components/interfaces'
+
+export const useMhrInformation = () => {
+  // const {
+  //   getMhrRegistrationHomeDescription,
+  //   getMhrRegistrationSubmittingParty,
+  //   getMhrRegistrationDocumentId,
+  //   getMhrAttentionReferenceNum,
+  //   getMhrRegistrationLocation,
+  //   getMhrRegistrationHomeOwnerGroups,
+  //   getStaffPayment
+  // } = useGetters<any>([
+  //   'getMhrRegistrationHomeDescription',
+  //   'getMhrRegistrationSubmittingParty',
+  //   'getMhrRegistrationDocumentId',
+  //   'getMhrAttentionReferenceNum',
+  //   'getMhrRegistrationLocation',
+  //   'getMhrRegistrationHomeOwnerGroups',
+  //   'getCertifyInformation',
+  //   'getStaffPayment'
+  // ])
+
```

Comment:
I am assuming this code will be enabled as more features are added?

Reply:
You bet, thanks! Forgot to push my clean up commit 👍 
Pushed now!